bigtable-dataflow-parent/bigtable-dataflow-seqio/README.md

```diff
@@ -5,14 +5,14 @@ Cloud Dataflow.
 
 ## Instructions
 
-Download [the import/export jar](http://search.maven.org/remotecontent?filepath=com/google/cloud/bigtable/bigtable-dataflow-import/0.9.5.1/bigtable-dataflow-import-0.9.5.1-shaded.jar), which is an aggregation of all required jars.
+Download [the import/export jar](http://search.maven.org/remotecontent?filepath=com/google/cloud/bigtable/bigtable-dataflow-seqio/0.9.6/bigtable-dataflow-import-0.9.6-shaded.jar), which is an aggregation of all required jars.
```

Comment:
Let's update the docs in a later PR.  The 0.9.5.1 instructions will still work, while the 0.9.6 will not work yet.